common/ownership_voucher/ownership_voucher.go

```diff
@@ -0,0 +1,92 @@
+// Package ownershipvoucher provides helper functions for generating, parsing and verifying ownership vouchers.
```

Comment:
Files need to begin with a license

// Copyright 2023 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

Reply:
Done.